umibukela/templates/partners.html

```diff
@@ -43,24 +43,14 @@
             <th>Province</th>
           </tr>
         </thead>
+{% for partner in partners %}
         <tr>
-          <td>Tshedza Development Project</td>
+          <td><a href="{% url 'partner' partner.slug %}">{{ partner.short_name }}</a></td>
+          <td><a href="{% url 'site-result' site_name='folweni' result_id='4' %}"></a></td>
           <td></td>
-          <td>Social Development</td>
-          <td>Gauteng</td>
```

Comment:
Should we leave the sector and province in for now, as a reminder that they should be included here? They could all be the same.